### DISCUSSION TITLE
Math size styles issue

### DISCUSSION THREAD

**Discussion by arkadiuszwojcik** [typst/typst#4120] [General]
### Description

I have following typst markup:

```
#box[
  $T_i/2 = display(T_i/2)$
]

#box[
  $T_i = display(T_i)$
]
```
that renders to:

![Screenshot 2024-05-12 174832](https://github.com/typst/typst/assets/2368976/fc7e140a-c36f-4aeb-b0fd-13e163b7c7a9)

In first box elements from left side are smaller than those from right side as explected. But in second box when fraction part is not present elements on both side are same in size. Is it correct behaviour? I would expect them to be different in size like in first case.

### Operating system

Windows

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
This is to be expected, because just T_i isn't large enough to be cramped inline, while the fraction is.
